AK/BuiltinWrappers.h

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2021, Nick Johnson <sylvyrfysh@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Concepts.h"
+#include "Platform.h"
+
+template<Unsigned IntType>
+inline constexpr int popcount(IntType value)
+{
+#if defined(__GNUC__) || defined(__clang__)
+    static_assert(sizeof(IntType) <= sizeof(unsigned long long));
+    if constexpr (sizeof(IntType) <= sizeof(unsigned int))
+        return __builtin_popcount(value);
+    if constexpr (sizeof(IntType) == sizeof(unsigned long))
+        return __builtin_popcountl(value);
+    if constexpr (sizeof(IntType) == sizeof(unsigned long long))
+        return __builtin_popcountll(value);
+    VERIFY_NOT_REACHED();
+#else
+    int ones = 0;
+    for (size_t i = 0; i < 8 * sizeof(IntType); ++i) {
+        if ((val >> i) & 1) {
+            ++ones;
+        }
+    }
+    return ones;
+#endif
+}
+
+// The function will return the number of trailing zeroes in the type. If
+//     the given number if zero, this function may contain undefined
```

Comment:
Typo: "… given number *is* zero …"

I think this intra-comment indentation is somewhat unusual. Could you remove the additional spaces?
E.g.
```
//     the …
```
to:
```
// the …
```

Reply:
Done! And nice catch on the typo!